lib/clusters/metering.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const Cluster = require('../Cluster');
+const { ZCLDataTypes } = require('../zclTypes');
+
+const ATTRIBUTES = {
+  currentSummationDelivered: { id: 0, type: ZCLDataTypes.uint48 },
+  currentSummationReceived: { id: 1, type: ZCLDataTypes.uint48 },
+  currentMaxDemandDelivered: { id: 2, type: ZCLDataTypes.uint48 },
+  currentMaxDemandReceived: { id: 3, type: ZCLDataTypes.uint48 },
+  dailyFreezeTime: { id: 5, type: ZCLDataTypes.uint16 },
+  powerFactor: { id: 6, type: ZCLDataTypes.int8 },
+  // CurrentMaxDemandDeliveredTime: { id: 8, type: ZCLDataTypes.UTC },
+  // CurrentMaxDemandReceivedTime: { id: 9, type: ZCLDataTypes.UTC },
+  profileIntervalPeriod: { id: 15, type: ZCLDataTypes.enum8 },
+  status: { id: 512, type: ZCLDataTypes.map8 },
+  unitofMeasure: { id: 768, type: ZCLDataTypes.enum8 },
+  multiplier: { id: 769, type: ZCLDataTypes.uint24 },
+  divisor: { id: 770, type: ZCLDataTypes.uint24 },
+  summationFormatting: { id: 771, type: ZCLDataTypes.map8 },
+  demandFormatting: { id: 771, type: ZCLDataTypes.map8 },
+  meteringDeviceType: { id: 773, type: ZCLDataTypes.map8 },
```

Comment:
Are you sure about the ids in this cluster? `meteringDeviceType` should have id `0x0306 === 774`. Please check all the ids very carefully, once this is merged it is very difficult to spot bugs here.

Reply:
I'll re-check them; I would suspect that a commit was not pushed to the PR

Reply:
removed